sources/models/src/modeled_types/oci_defaults.rs

```diff
@@ -94,15 +94,45 @@ mod oci_defaults_capabilities {
 #[derive(Debug, Copy, Clone, Eq, PartialEq, Hash, Scalar, Serialize, Deserialize)]
 #[serde(rename_all = "kebab-case")]
 pub enum OciDefaultsResourceLimitType {
+    MaxAddressSpace,
+    MaxCoreFileSize,
+    MaxCpuTime,
+    MaxDataSize,
+    MaxFileLocks,
+    MaxFileSize,
+    MaxLockedMemory,
+    MaxMsgqueueSize,
+    MaxNicePriority,
     MaxOpenFiles,
+    MaxPendingSignals,
+    MaxProcesses,
+    MaxRealtimePriority,
+    MaxRealtimeTimeout,
+    MaxResidentSet,
+    MaxStackSize,
 }
 
 // We are leaving this implementation open for the easy addition of
 // future resource limits.
 impl OciDefaultsResourceLimitType {
     pub fn to_linux_string(&self) -> &'static str {
         match self {
+            OciDefaultsResourceLimitType::MaxAddressSpace => "RLIMIT_AS",
```

Comment:
nit: you can clean up some of the repetition with a `use` statement:
```suggestion
    pub fn to_linux_string(&self) -> &'static str {
        use OciDefaultsResourceLimitType::*;
        match self {
            MaxAddressSpace => "RLIMIT_AS",
```